plutus-ledger-api/src/Plutus/V1/Ledger/Scripts.hs

```diff
@@ -407,3 +409,5 @@ makeLift ''RedeemerHash
 makeLift ''Datum
 
 makeLift ''Redeemer
+
+makeWrapped ''Script
```

Comment:
I would prefer to use `deriving anyclass Wrapped` instead, but it complains about deducing profunctor, whereas the TH works... :shrug: